frontend/templates/views/partials/content.j2

```diff
@@ -8,10 +8,23 @@
 
   {% if doc.title and doc.view in ['/views/detail/docs-detail.j2', ] %}
   <h1>{{ doc.title }}</h1>
-  {% endif %}
 
+  {% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
```

Comment:
This css should only be included, if the tip is shown.
Should be possible to move this line into the if block.

Reply:
Hm, thought about this as well: wouldn't help. The `[% if %]` only get's evaluated server-side when the CSS has long been combined.

But you're right currently this PR would add the tip styling to *every* docs detail page.

Reply:
Ok, I did not pay attention to square or curly brackets at first :-o

The only thing we could do is to only include the css if not all formats are supported.
But most of the guides are not for all formats, so that would not help much.

Another thing is, that the changes do not work for components since they use another template (component-detail.j2). I think that should be added.

I would not add the layer for examples, since we cannot specify multiple formats, but lots of examples are in general also correct for other formats (see the components section where every example is only visible for websites)